src/output.c

```diff
@@ -27,30 +27,333 @@
 #include "private.h"
 #include "unicode.h"
 
-void output_push(output_t *st, uint8_t *pkt, unsigned int len, unsigned int program, unsigned int stream_id)
+#define RADIO_FRAME_SAMPLES_FM (NRSC5_AUDIO_FRAME_SAMPLES * 135 / 8)
+#define RADIO_FRAME_SAMPLES_AM (NRSC5_AUDIO_FRAME_SAMPLES * 135 / 256)
```

Comment:
AM seems to be broken; I get lots of "Internal writer clock bug. Full of samples". The intermediate sample rate for AM is 1/16 of the rate for FM, so this change seems to fix the problem:
```suggestion
#define RADIO_FRAME_SAMPLES_AM (NRSC5_AUDIO_FRAME_SAMPLES * 135 / 128)
```

Reply:
Thanks for testing AM. You're correct. I made an error reducing fractions.